packages/blocks/src/api/factory.js

```diff
@@ -94,62 +124,45 @@ export function createBlocksFromInnerBlocksTemplate(
 }
 
 /**
- * Given a block object, returns a copy of the block object while sanitizing its attributes,
- * optionally merging new attributes and/or replacing its inner blocks.
+ * Given a block object, returns a copy of the block object,
+ * optionally merging new attributes, replacing its inner blocks, and/or
+ * filtering out attributes which do not have copy support.
  *
- * @param {Object} block           Block instance.
- * @param {Object} mergeAttributes Block attributes.
- * @param {?Array} newInnerBlocks  Nested blocks.
+ * @param {Object}   block                        Block instance.
+ * @param {Object}   mergeAttributes              Block attributes.
+ * @param {?Array}   newInnerBlocks               Nested blocks.
+ * @param {?Object}  options                      Cloning options.
```

Comment:
I wonder if it makes sense to mark this as `__experimental`. It would be easy to miss unless the developer calling the function was looking closely at the docs. At any rate, we have until WP 6.0 to decide to remove or change it.